uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/controllers/request-ctrl.js

```diff
@@ -1027,7 +1029,12 @@ define([
           // When in Admin Dashboard
           return Contact.all()
             .then(function (contacts) {
-              this.managedContacts = contacts.list;
+              var self = this;
+
+              this.managedContacts = _.remove(contacts.list, function (contact) {
+                // Removes the manager/admin from the list of managees
+                return contact.id !== self.directiveOptions.contactId;
+              });
```

Comment:
`}.bind(this));` is being used above, it should be used here instead of `var self = this`, etc.